gdc_client/parcel/segment.py

```diff
@@ -30,6 +28,17 @@
 )
 from gdc_client.parcel.const import SAVE_INTERVAL
 
+if OS_WINDOWS:
+    WINDOWS = True
+    from six.moves.queue import Queue
```

Comment:
Still need to remove the `six` reference here

Reply:
From my understanding, the only reason the WINDOWS variable is used in this file is to decide whether to use the Queue from six.moves.queue or use the Queue from multiprocessing.Manager. 

I think we should remove them or keep them altogether. Could you please double check this?

Reply:
And also, I can not understand why to use another `WINDOWS` variable when `OS_WINDOWS` is already used. It feels really wired for me to add the `WINDOWS` variable here.

If you think it makes sense to you, I will remove the six and keep the rest. I might be too new here to see the whole picture.

Reply:
The purpose of the ticket is to remove the usage of `six`.  Limit your changes to that.  There are many places in this code base where we could make improvements.  Those are out-of-scope of this ticket.

I am not suggesting to delete this line of code entirely.  Instead, you should remove the use of `six` here and just use py3:
```
from queue import Queue
```

By doing this, we make a very straight forward change that should be 100% compatible with the existing functionality.

Reply:
Thanks, it makes sense.